### ISSUE TITLE
build warning with master branch and Sphinx 2.1.2

### ISSUE THREAD

**Issue by williehallock802** [rst2pdf/rst2pdf#804]
```
$ sphinx-build -b pdf source build/pdf

Running Sphinx v2.1.2
.../python3.6/site-packages/sphinx/registry.py:473: RemovedInSphinx30Warning: sphinx.environment.NoUri is deprecated. Check CHANGES for Sphinx API modifications.
  mod = __import__(extname, None, None, ['setup'])
```

I am using commit 25713408.

**Comment by johnthagen** [comment]
Related to #801

**Comment by johnthagen** [comment]
See #788

**Comment by akrabat** [comment]
I believe that this is fixed as #788 is now merged.
